### DISCUSSION TITLE
Stars count is wrong during the day

### DISCUSSION THREAD

**Discussion by gingerbreadassassin** [aaronwmorris/indi-allsky#1482] [General]
I'm pretty sure there's at least _one_ pictured here...
![361495452-1e1afb6c-85f1-4057-86fd-614b5fcf4a83](https://github.com/user-attachments/assets/595ae676-c03c-4230-92fb-c5d6988a6d20)

**Comment by aaronwmorris** [comment]
I will admit... I spent way too much time trying to find the single star in the picture.  :-)  Bravo

**Reply by gingerbreadassassin** [reply]
It might've been a bit more obvious without the cloud, but I just couldn't wait to share :D
